src/core/class-utils.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Checks if a class contains a method either itself or in it's inheritance chain
+ *
+ * @param {Function} Class - The class to check
+ * @param {string} method - The name of the method to check
+ * @returns {boolean} if a valid class and contains the method in it's inheritance chain
+ * @ignore
+ */
+export const classHasMethod = (Class, method) => {
```

Comment:
typically a parameters start with lower-case .. I wonder if there's some good replacement for `Class` starting with a lower case? maybe `possibleClass` or 'testClass`?

Reply:
Yep agreed, I think `testClass` makes sense. Have updated this